### DISCUSSION TITLE
Web Searches plugin - How do I make it comptible with Application URI Schemes like previous version?

### DISCUSSION THREAD

**Discussion by silic0n64** [Flow-Launcher/Flow.Launcher#818] [Q&A] [answered]
In the previous version of Web Searches plugin where there were no option to start web browser from a location, I was able to perform opeartions like opening and searching an external application using URL: `obsidian://search?vault=vault-namd&query={q}` alongside regular google searches using URL: `https://www.google.com/search?q={q}`

But in the updated plugin, with default(blank) browser starting location set, application URi schemes don't work, it just opens default browser and does nothing. If I add the application path as the starting location for browser, then the URI scheme works, opens the application like it should, but then the google search doesn't work(only opens the application, instead of browser),  It was perfect in the previous version of the plugin. I mainly use Wox for this type of URI schemes and it's keeping me from moving to the beautiful Flow-launcher. What can I do to achieve what I want?

**Comment by taooceros** [comment]
I will test that out later today. I can reproduce your issue.

**Reply by onesounds** [reply]
@taooceros not yet?

**Reply by taooceros** [reply]
> @taooceros not yet?

What happened to me😱? I have already test that out and we search plugin can do that with Firefox but not edge. If you still want to use edge, contact @Garulf. He has written a plugin for obsidian url that can handles the situation.

Sorry for the late reply.

**Comment by onesounds** [comment]
obsidian plugin is release. check the plugin store please.

**Comment by taooceros** [comment]
I guess we can support application uri by checking whether the link is for http before access it.

**Comment by taooceros** [comment, marked as answer]
I make it works now #968
